CHANGELOG

```diff
@@ -1,6 +1,7 @@
 2.8.6.dev1
 ----------
 
+- fix #1259: allow for double nodeids in junitxml
```

Comment:
I think it would be better to mention the regression here, for example: "Fix regression introduced in 2.8.5 about junitxml raising an error on duplicated node ids" or something similar